metrics/README.md

```diff
@@ -0,0 +1,60 @@
+# Clear Containers metrics tests
+
+This is a collection of scripts designed to measure performance and responsiveness
+of the different components which form Clear Containers.
+
+## Metrics tests - content
+
+The tests are organized in the following sections:
+
+| Section   | Short description
+| ----------| --------------------------------------------------------------------------------
+| density   | Masure the amount of memory used by N containers under different configurations.
+| network   | Measere a set of network essentials such as: bandwidth, jitter, latency etc.
+| storage   | Measure storage bandwith using different configurations.
+| time      | Measure the latency that a container[s] take[s] to do a determined workload.
+
+## HOWTO execute metrics scripts
+
+This section includes a helper script `run_all_metrics.sh`. It allows to execute all metrics
+tests or only a specific section.
+
+## Options
+
+if no options are specified, `run_all_metrics.sh` will execute all tests by default.
+
+### Only run the metrics tests from time section
+```
+-l, --latency  Run latency/time metrics tests
+```
+
+### Only run the metrics tests from density section
+```
+-m,  --memory  Run memory metrics tests    path to TOML configuration file
+```
+
+### Only run the metrics tests from network section
+```
+-n, --network  Run network metrics tests
+```
+
+### Only run the metrics tests from storage section
+```
+-s, --storage  Run I/O storage metrics tests
+```
+
+### Help
+```
+-h, --help     show help
+```
+
+## Usage
+```bash
+$ sudo ./run_all_metrics.sh [options]
```

Comment:
You could note somewhere appropriate (maybe new section near the top, or a footnote) that some tests require root access to run correctly. The density tests do for using `smem` to inspect the system components for instance. I'm not sure if other tests do from memory (I have a feeling network does not for instance).